msgpack/__init__.py

```diff
@@ -3,7 +3,12 @@
 from .exceptions import *
 
 from collections import namedtuple
+from numbers import Number, Integral
```

Comment:
Don't use them.  Just use int.

Reply:
May I ask why not? As a user I find it obnoxious to have to convert NumPy types to pure Python types and the `Number` and `Integral` general types alleviate that.

Reply:
> May I ask why not? 

### Consistency

We don't use them in normal integers.  Why do we support it in Timestamp?
It doesn't make sense at all.

### Performance

```
$ python3 -m timeit -s 'from numbers import Integral' -- 'isinstance(42, Integral)'
500000 loops, best of 5: 443 nsec per loop
$ python3 -m timeit 'isinstance(42, int)'
5000000 loops, best of 5: 59.9 nsec per loop
```

---

>  As a user I find it obnoxious to have to convert NumPy types to pure Python types

Do you really want to use numpy types in Timestamp????

Reply:
Thanks, everything you said makes sense.